packages/material-ui/src/CardMedia/CardMedia.js

```diff
@@ -11,9 +11,12 @@ export const styles = {
     backgroundRepeat: 'no-repeat',
     backgroundPosition: 'center',
   },
-  /* Styles applied to the root element if `component="video, audio, picture, iframe, or img"`. */
+  /* Styles applied to the root element if `component="video, audio or iframe"`. */
```

Comment:
This is still set if component is a picture or img

Reply:
My bad! I'll correct it.